bodhi/server/services/updates.py

```diff
@@ -200,10 +200,10 @@ def set_request(request):
     try:
         update.set_request(request.db, action, request.user.name)
     except BodhiException as e:
-        log.exception("Failed to set the request")
+        log.error("Failed to set the request")
         request.errors.add('body', 'request', str(e))
     except Exception as e:
-        log.exception("Unhandled exception in set_request")
+        log.error("Unhandled exception in set_request")
```

Comment:
We do want to log an exception when an unhandled exception occurs.

```suggestion
        log.exception("Unhandled exception in set_request")
```